apps/pipe-intermediate/src/shared/wrapFn.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+type CallbackFunction = (arg1: any, arg2: any) => string;
+@Pipe({ name: 'wrapFn', standalone: true })
+export class WrapFunction implements PipeTransform {
+  transform(
+    callback: CallbackFunction,
+    arg1: string | number,
+    arg2: number | boolean
+  ) {
+    return callback(arg1 as string | number, arg2 as number | boolean);
+  }
+}
```

Comment:
Sorry but that's not very flexible. What if the function has 4 parameters ? What if the first one is an object or a boolean, or anything else ? 

this challenge is a big more complex. You should look at Typescript utility types to create more generic types